master does not compile

I cannot seem to get Master at  36eb892
to compile.  I wonder if a change in the adam snapshot dependency broke things, as I am fairly sure I compiled Mango successfully early last week since the latest commit.

'''
paschalj@big1:~/mango/v1/mango$ mvn clean package -DskipTests
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] mango: an ADAM-based genomics visualization toolkit
[INFO] mango-core: an ADAM-based genomics visualization toolkit
[INFO] mango-cli: an ADAM-based genomics visualization toolkit
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building mango: an ADAM-based genomics visualization toolkit 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ mango-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ mango-parent ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ mango-parent ---
[INFO] 
[INFO] --- scalariform-maven-plugin:0.1.4:format (default) @ mango-parent ---
[INFO] Modified 0 of 27 .scala files
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:compile (scala-compile-first) @ mango-parent ---
[INFO] No sources to compile
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:testCompile (scala-test-compile-first) @ mango-parent ---
[INFO] No sources to compile
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building mango-core: an ADAM-based genomics visualization toolkit 0.0.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:3.0.0:clean (default-clean) @ mango-core ---
[INFO] Deleting /home/paschalj/mango/v1/mango/mango-core/target
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ mango-core ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ mango-core ---
[INFO] 
[INFO] --- scalariform-maven-plugin:0.1.4:format (default) @ mango-core ---
[INFO] Modified 0 of 24 .scala files
[INFO] 
[INFO] --- maven-resources-plugin:3.0.1:resources (default-resources) @ mango-core ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /home/paschalj/mango/v1/mango/mango-core/src/main/resources
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:compile (scala-compile-first) @ mango-core ---
[WARNING]  Expected all dependencies to require Scala version: 2.10.6
[WARNING]  net.liftweb:lift-json_2.10:2.6.2 requires scala version: 2.10.4
[WARNING] Multiple versions of scala libraries detected!
[INFO] /home/paschalj/mango/v1/mango/mango-core/src/main/scala:-1: info: compiling
[INFO] Compiling 13 source files to /home/paschalj/mango/v1/mango/mango-core/target/scala-2.10.6/classes at 1489287951396
[ERROR] /home/paschalj/mango/v1/mango/mango-core/src/main/scala/org/bdgenomics/mango/models/LazyMaterialization.scala:216: error: type mismatch;
[ERROR]  found   : org.apache.spark.rdd.RDD[(org.bdgenomics.adam.models.ReferenceRegion, (String, T))]
[ERROR]  required: Array[(org.bdgenomics.adam.models.ReferenceRegion, (String, T))]
[ERROR]         intRDD = intRDD.multiput(data)
[ERROR]                                  ^
[ERROR] one error found
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] mango: an ADAM-based genomics visualization toolkit  SUCCESS [  2.441 s]
[INFO] mango-core: an ADAM-based genomics visualization toolkit FAILURE [  6.801 s]
[INFO] mango-cli: an ADAM-based genomics visualization toolkit SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 9.442 s
[INFO] Finished at: 2017-03-11T22:05:55-05:00
[INFO] Final Memory: 37M/1726M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile (scala-compile-first) on project mango-core: wrap: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :mango-core

'''

--- a/mango-core/src/main/scala/org/bdgenomics/mango/models/LazyMaterialization.scala
+++ b/mango-core/src/main/scala/org/bdgenomics/mango/models/LazyMaterialization.scala
@@ -213,7 +213,7 @@ abstract class LazyMaterialization[T: ClassTag](name: String,
         intRDD.persist(StorageLevel.MEMORY_AND_DISK)
       } else {
         val t = intRDD
-        intRDD = intRDD.multiput(data)
+        intRDD = intRDD.multiputRDD(data)
         t.unpersist(true)
         intRDD.persist(StorageLevel.MEMORY_AND_DISK)
       }

